non-wheel/active-jobs-dashboard/root.tpl

```diff
@@ -0,0 +1,64 @@
+<html>
+<head><title>Active Job Dashboard</title>
+<script type="text/javascript" src="jquery-latest.js"></script>
+<script type="text/javascript" src="jquery.tablesorter.min.js"></script>
```

Comment:
Yes indeed, this will make it much more usable. Should be able to "git clone" and go!